schemas/JSON/manifests/v1.1.0/manifest.defaultLocale.1.1.0.json

```diff
@@ -0,0 +1,171 @@
+{
+  "$id": "https://aka.ms/winget-manifest.defaultlocale.1.1.1.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "A representation of a multiple-file manifest representing a default app metadata in the OWC. v1.1.1",
+  "definitions": {
+    "Url": {
+      "type": [ "string", "null" ],
+      "pattern": "^([Hh][Tt][Tt][Pp][Ss]?)://.+$",
+      "maxLength": 2000,
+      "description": "Optional Url type"
+    },
+    "Tag": {
+      "type": [ "string", "null" ],
+      "minLength": 1,
+      "maxLength": 40,
+      "description": "Package moniker or tag"
+    },
+    "Agreement": {
+      "type": "object",
+      "properties": {
+        "AgreementLabel": {
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 100,
+          "description": "The label of the Agreement. i.e. EULA, AgeRating, etc. This field should be localized. Either Agreement or AgreementUrl is required. When we show the agreements, we would Bold the AgreementLabel"
+        },
+        "Agreement": {
+          "type": [ "string", "null" ],
+          "minLength": 1,
+          "maxLength": 10000,
+          "description": "The agreement text content."
+        },
+        "AgreementUrl": {
+          "$ref": "#/definitions/Url",
+          "description": "The agreement URL."
+        }
+      },
+      "minProperties": 1

```

Comment:
>"minProperties": 1 [](http://example.com/codeflow?start=6&length=18)

If we really want to be strict, this would better be in ManifestValidation method